files/clamav-daemon.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Clam AntiVirus userspace daemon
+Documentation=man:clamd(8) man:clamd.conf(5) http://www.clamav.net/lang/en/doc/
+Requires=clamav-daemon.socket
+# Check for database existence
+ConditionPathExistsGlob=/var/lib/clamav/main.{c[vl]d,inc}
+ConditionPathExistsGlob=/var/lib/clamav/daily.{c[vl]d,inc}
+
+[Service]
+ExecStart=/usr/sbin/clamd --foreground=true
+# Reload the database
+ExecReload=/bin/kill -USR2 $MAINPID
+StandardOutput=syslog
+Nice=18
```

Comment:
Niceness values don't affect whether or not a job or service runs;

If a person is compiling something then that will take priority, otherwise the scan of the root filesystem has 5 hours to run (assuming people work 9-5)...